rat-pac-jsns2 compile error on os x

- Working on OS X with root5.34.36/Geant4.10.01.03 
- after coning and scons, terminal shows messages as below:

src/physics/PhysicsList.cc:162:3: error: instantiation of variable
      'G4VUPLSplitter<G4VUPLData>::offset' required here, but no definition is
      available [-Werror,-Wundefined-var-template]
  theParticleIterator->reset();
  ^
/opt/geant4/bin/../include/Geant4/G4VUserPhysicsList.hh:153:56: note: expanded
      from macro 'theParticleIterator'
#define theParticleIterator ((this->subInstanceManager.offset[this->g4vu...
                                                       ^
/opt/geant4/bin/../include/Geant4/G4VUPLSplitter.hh:195:38: note: forward
      declaration of template entity is here
    G4RUN_DLL G4ThreadLocalStatic T* offset; //Pointer to first instanc...
                                     ^
1 error generated.
scons: *** [build/macosx64/physics/PhysicsList.o] Error 1
-same thing is also occurred in rat-pac compile.

--- a/config/ARCH.macosx64
+++ b/config/ARCH.macosx64
@@ -19,4 +19,5 @@ if len(parts) >= 2 and int(parts[1]) >= 8:
 
 arch['Xm'].update( { 'CPPPATH' : '/sw/include',
                      'LIBPATH' : '/sw/lib' }    )
+arch['default']['CXXFLAGS'] += ['-Wno-reserved-id-macro', '-Wno-keyword-macro']
 Return('arch')

--- a/src/physics/PhysicsList.cc
+++ b/src/physics/PhysicsList.cc
@@ -23,6 +23,9 @@
 #include <RAT/PhysicsListMessenger.hh>
 #include <RAT/PhysicsList.hh>
 
+template<>
+G4VUPLData* G4VUPLSplitter<G4VUPLData>::offset = NULL;
+
 namespace RAT {
 
 PhysicsList::PhysicsList() : Shielding(), wlsModel(NULL) {

